cli.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+'use strict';
+
+var path = require('path');
+var childProcess = require('child_process');
+var globby = require('globby');
+var inquirer = require('inquirer');
+var assign = require('lodash.assign');
+var utils = require('./cli-utils');
+var codemods = require('./codemods.json');
+
+function runScripts(scripts, files) {
+	var binPath = require.resolve('jscodeshift/bin/jscodeshift.sh');
+	var spawnOptions = {
+		env: assign({}, process.env, {PATH: path.resolve('node_modules/.bin') + ':' + process.env.PATH}),
+		stdio: 'inherit'
+	};
+
+	var result;
+	scripts.forEach(function (script) {
+		result = childProcess.spawnSync(binPath, ['-t', script].concat(files.split(' ')), spawnOptions);
```

Comment:
Yeah but you can only do that once right? Since `jscodeshift` only accepts one transformer file at a time (afaik), we need to spawn it multiples times.

But I agree that it would be nice to avoid spawning it...